core/quill.ts

```diff
@@ -705,12 +744,13 @@ function expandConfig(
         uploader: true,
       },
     },
-    userConfig,
+    { ...omit(userConfig, 'theme') },
```

Comment:
Why we omit `theme`?

Reply:
Yeah I think that makes sense but we should not mix code changes with type changes to make the PR easier to review. Especially let's avoid add any dependencies in this PR.